File: examples/csharp/Search1.cs
using System;

class Search1
{
    static void Main()
    {
        int[] arr = { 5, 2, 8, 1, 9, 3 };
        int target = 8;
        int index = LinearSearch(arr, target);
        Console.WriteLine(index >= 0 ? $"Found at index {index}" : "Not found");
    }

    static int LinearSearch(int[] arr, int target)
    {
        for (int i = 0; i < arr.Length; i++)
            if (arr[i] == target) return i;
        return -1;
    }
}

File: examples/java/Search1.java
public class Search1 {
    static int linearSearch(int[] arr, int key) {
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] == key) return i;
        }
        return -1;
    }

    public static void main(String[] args) {
        int[] arr = {5, 12, 8, 3, 19, 7};
        int key = 8;
        int idx = linearSearch(arr, key);
        System.out.println(idx >= 0 ? "Found at " + idx : "Not found");
    }
}
